### ISSUE TITLE
FR: Add `fast_poll` option for failed ping checks

### ISSUE THREAD

**Issue by thezackm** [kentik/ktranslate#612]
As a user, I need to quickly validate potential outages on my devices. In the current setup, the `ping` tests are scheduled on the same interval as the `poll_time_sec`. This means that I potentially need to wait 5+ minutes to validate a "down" device.

Request that functionality is added that follows the following logic:

> When packet loss = 100%, initiate 'fast polling' where we continue to send ping tests every 10s, for a total of 120s to validate a down condition.

This should only apply when `response_time: true | ping_only: true`. A nice to have would be the ability to control the interval and overall duration as well.
